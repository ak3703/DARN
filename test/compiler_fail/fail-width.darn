int main() {
	int[5] a;
	width(a);
}
