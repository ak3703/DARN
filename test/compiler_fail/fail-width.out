Fatal error: exception Failure("cannot get the width of non-2d-matrix")
